### PR TITLE
Add YouTube statistics dashboard script

### PR DESCRIPTION
**Pull request by astrit**
## Description

Get your channel statistics like: views, subscribers and videos on Raycast dashboard
## Type of change

- [x] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [ ] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

youtube-statistics

## Dependencies / Requirements

- You need to specify `Channel ID` — [Guide](https://support.google.com/youtube/answer/3250431?hl=en)
- You need to specify `YouTube Data API v3 API Key` — [Guide](https://developers.google.com/youtube/registering_an_application)

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/dashboard/youtube-statistics.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Youtube Statistics
```
Comment (unnamedd) [inline commands/dashboard/youtube-statistics.rb:5]:
```suggestion
# @raycast.title YouTube Statistics
```

File: commands/dashboard/youtube-statistics.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# @raycast.mode inline
+# @raycast.refreshTime 1h
+
+# Optional parameters:
+# @raycast.packageName Youtube Statistics
```
Comment (unnamedd) [inline commands/dashboard/youtube-statistics.rb:10]:
```suggestion
# @raycast.packageName YouTube Statistics
```

File: commands/dashboard/youtube-statistics.rb [hunk 3]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
```
Comment (unnamedd) [inline commands/dashboard/youtube-statistics.rb:1]: As you need extra setup to make this Script Command work, you need to rename this file to: `youtube-statistics.template.rb`.

File: commands/system/wifi-password.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (unnamedd) [inline commands/system/wifi-password.sh:1]: This Script Command doesn’t belong to this pull request.

**Review by unnamedd — CHANGES_REQUESTED** [review]
Hi @astrit, thanks for another contribution. 
I made some few comments, please, after address them, let me know!

File: commands/system/wifi-password.sh (hunk 4 above)
Comment (astrit) [reply commands/system/wifi-password.sh]: Do you mind checking now ?

**Comment by astrit** [comment]
@unnamedd I did update and if you don't mind checking it

**Pull request closed without merging**